kong/clustering/tls.lua

```diff
@@ -189,7 +189,7 @@ end
 ---@param cp_cert      kong.clustering.certinfo # clustering certinfo table
 ---@param dp_cert_pem  string                   # data plane cert text
 ---
----@return boolean? success
+---@return table|nil x509 instance
 ---@return string?  error
```

Comment:
"x509 instance" need to be aligned with "error" message like

```
---@return table|nil x509 instance
---@return string?   error
```

Reply:
done